src/FixerFactory.php

```diff
@@ -160,8 +160,8 @@ public function useRuleSet(RuleSetInterface $ruleSet): self
 
             if (null !== $config) {
                 if ($fixer instanceof ConfigurableFixerInterface) {
-                    if (\count($config) < 1) {
-                        throw new InvalidFixerConfigurationException($fixer->getName(), 'Configuration must be an array and may not be empty.');
+                    if (\count($config) === 0) {
```

Comment:
tbh, i don't see a value in change.
why it was "failing" ?

Reply:
I've made this during conflict solving, reverted for now.

Personally I'd apply POLA here checking for `=== 0` is much more expected than `< 1`.